sql/CMakeLists.txt

```diff
@@ -120,7 +120,7 @@ DTRACE_INSTRUMENT(sql)
 TARGET_LINK_LIBRARIES(sql ${MYSQLD_STATIC_PLUGIN_LIBS} 
   mysys mysys_ssl dbug strings vio pcre ${LIBJEMALLOC}
   ${LIBWRAP} ${LIBCRYPT} ${LIBDL} ${CMAKE_THREAD_LIBS_INIT}
-  ${SSL_LIBRARIES})
+  ${LIBSYSTEMD} ${SSL_LIBRARIES})
```

Comment:
this is ./src/libsystemd/sd-daemon/sd-daemon.c in the systemd source which also has dependencies on ./src/basic/path-util.h ./src/basic/socket-util.h ./src/basic/util.h.

Perhaps just setting 10.1 as a target.